examples/ssd/caffe2npz.py

```diff
@@ -59,17 +61,18 @@ def add_link(self, name, link):
                 data = data.reshape((-1, 4) + data.shape[1:])
                 data[:, [1, 0, 3, 2]] = data.copy()
             print('{:s} -> {:s} (xy -> yx)'.format(name, new_name))
-        else:
+        elif match:
```

Comment:
I see. How about checking `self.within_init_scope` (https://github.com/chainer/chainer/blob/master/chainer/link.py#L153)? This property becomes `True` if the attribute is a layer (a child link).